flink-state-backends/flink-statebackend-forst/src/test/java/org/apache/flink/state/forst/ForStStateBackendV2Test.java

```diff
@@ -90,9 +91,13 @@ protected ConfigurableStateBackend getStateBackend() throws Exception {
         Configuration config = new Configuration();
         if (hasLocalDir) {
             config.set(LOCAL_DIRECTORIES, tempFolderForForStLocal.toString());
+            config.set(USE_INGEST_DB_RESTORE_MODE, true);
         }
         if (hasRemoteDir) {
             config.set(REMOTE_DIRECTORY, tempFolderForForstRemote.toString());
+            // Async state + remote + ingestDB is not supported, because ingestDB needs to link
+            // files.
+            config.set(USE_INGEST_DB_RESTORE_MODE, false);
```

Comment:
So IIUC only ingest db is tested, what if the delete range?

I'd suggest parameterized related IT cases

Reply:
👍 Thanks for the suggestion, there are too many combinations, I only chose several tests.